### PR TITLE
[16.0][OU-ADD] pad: nothing to do

### PR DESCRIPTION
**Pull request by daiduongnguyen-odoo**
Obsolete module, use directly in odoo itself

### CODE REVIEW

**Comment by legalsylvain** [comment]
Hi @duong77476 . thanks for your contribution. In whitch module the feature is merged ?

**Comment by daiduongnguyen-odoo** [comment]
> Hi @duong77476 . thanks for your contribution. In whitch module the feature is merged ?

Hmm via this PR `https://github.com/odoo/odoo/pull/75768` i think i would be `web_editor` . So in that case, should i write `Merged into web_editor` ?

**Comment by daiduongnguyen-odoo** [comment]
And here is the PR that delete it `https://github.com/odoo/odoo/pull/76467`

**Review by legalsylvain — CHANGES_REQUESTED** [review]
In that case, you should update the apriori.py file. (The merge section)

**Comment by daiduongnguyen-odoo** [comment]
> In that case, you should update the apriori.py file. (The merge section)

Done sir , thanks

**Review by legalsylvain — APPROVED** [review]

File: docsource/modules150-160.rst [hunk 1]
```diff
@@ -532,7 +532,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | |new| onboarding                                |                      |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| |del| pad                                       |                      |                                                 |
+| |del| pad                                       |Done                  |Obsolete module, use directly in odoo itself     |
```
Comment (MiquelRForgeFlow) [inline docsource/modules150-160.rst]:
```suggestion
| |del| pad                                       |Done                  | Merged into web_editor  |
```

File: openupgrade_scripts/apriori.py [hunk 2]
```diff
@@ -45,6 +45,7 @@
     "sale_project_account": "sale_project",
     "website_sale_delivery_giftcard": "website_sale_loyalty_delivery",
     "website_sale_gift_card": "website_sale_loyalty",
+    "pad": "web_editor",
```
Comment (MiquelRForgeFlow) [inline openupgrade_scripts/apriori.py]: Please, put above pos_coupon to maintain alphabetical order.

File: docsource/modules150-160.rst (hunk 1 above)
Comment (daiduongnguyen-odoo) [reply docsource/modules150-160.rst]: Sure

File: openupgrade_scripts/apriori.py (hunk 2 above)
Comment (daiduongnguyen-odoo) [reply openupgrade_scripts/apriori.py]: Done sir, i'll take note for this for other relevant PR

**Pull request merged**